dev-env/docker-compose.yaml

```diff
@@ -0,0 +1,54 @@
+# SPDX-FileCopyrightText: 2024 SAP SE or an SAP affiliate company and Greenhouse contributors
+# SPDX-License-Identifier: Apache-2.0
+
+services:
+  envtest:
+    image: ghcr.io/cloudoperators/greenhouse-dev-env:main
+    volumes:
+      - ./envtest:/envtest
+      - ./webhook-certs:/webhook-certs
+    ports:
+      - "3005:8090"
+    environment:
+      # one of ["test-org-admin", "test-org-member"] - leave unset for "cluster-admin"
+      - DEV_ENV_CONTEXT
+      # set to true for control plane output
+      - KUBEBUILDER_ATTACH_CONTROL_PLANE_OUTPUT=false
+    healthcheck:
+      test: ["CMD", "cat", "/tmp/.envtest-running"]
+      interval: 10s
+      timeout: 10s
+      retries: 5
+  greenhouse:
+    image: ghcr.io/cloudoperators/greenhouse:main
+    environment:
+      - KUBECONFIG=/envtest/internal.kubeconfig
+      - HEADSCALE_API_URL=mock_empty
+      - HEADSCALE_API_KEY=mock_empty
+      - TAILSCALE_PROXY=mock_empty
+    volumes:
+      - ./envtest:/envtest
+      - ./webhook-certs:/tmp/k8s-webhook-server/serving-certs
+    depends_on:
+      envtest:
+        condition: service_healthy
+    network_mode: service:envtest
+    command: --dns-domain localhost
+  greenhouse-ui:
+    image: ghcr.io/cloudoperators/greenhouse-dev-ui:main
+    environment:
+      - PORT=3000
+    ports:
+      - "3000:3000"
+  bootstrap:
+    image: ghcr.io/cloudoperators/greenhouse-dev-env:main
+    environment:
+      - KUBECONFIG=/envtest/internal.kubeconfig
+    volumes:
+      - ./envtest:/envtest
+      - ./bootstrap:/bootstrap
+    command: kubectl apply -f /bootstrap
```

Comment:
Would it be possible to make the boostrapping configurable? e.g. always the empty test-org but make it optional to add the plugins and clusters? 
This would make testing a feature/plugin easier, as the controller manager logs are full of errors trying to reconcile the sample plugins & cluster.

Reply:
I updated the docs. Does that work for you?

Reply:
Thank you